checkstyle-tester/projects-to-test-on-3.properties

```diff
@@ -0,0 +1,28 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+
+jabref|git|https://github.com/JabRef/jabref|main|
+error-prone|git|https://github.com/google/error-prone|master|
+jpf-core|git|https://github.com/javapathfinder/jpf-core|master|
+weblogic-image-tool|git|https://github.com/oracle/weblogic-image-tool|main|
+react-native|git|https://github.com/facebook/react-native|main|
+weblogic-monitoring-exporter|git|https://github.com/oracle/weblogic-monitoring-exporter|main|
+nosql-java-sdk|git|https://github.com/oracle/nosql-java-sdk|main|
+java|git|https://github.com/kubernetes-client/java|master|
+spring-boot|git|https://github.com/spring-projects/spring-boot|main|
+spring-integration|git|https://github.com/spring-projects/spring-integration|main|
+spring-session|git|https://github.com/spring-projects/spring-session|main|
+spring-security|git|https://github.com/spring-projects/spring-security|main|
+RxJava|git|https://github.com/ReactiveX/RxJava|3.x|
```

Comment:
https://github.com/checkstyle/contribution/blob/ce7bad08ea5d8b2c1f18ec1e425436f2fc486e2f/checkstyle-tester/projects-to-test-on.properties#L38

Let's update existing place

Reply:
done